stanford-earth/SE3_BLT
# EARTH-260: Update package drupal-carvan-8.x-1.x

# NOT READY
- composer.lock appears to have made a number of changes unrelated to drupal-caravan.  I am curious to hear what I might be doing wrong with managing composer.lock.

# Summary
- This changes the branch name for drupal-caravan to dev-8.x-1.x, however, I would be interested to hear whether it should actually be something like ^8.x-1.  Or more specifically ^8.x.-1.0-alpha1.

# Needed By
- When testing on new machines, it is helpful for se3_blt to pull in the latest version of caravan that I am working on.

# Steps to Test

1. Run `composer update`.
2. Run `cp vendor/su-sws/drupal-caravan/default.caravan.yml caravan.yml`.
3. Modify `vendor/su-sws/drupal-caravan/caravan.yml` for your machine and project.
4. Run `blt sws:caravan`.
5. Wait an extremely long time.
6. Then you should be able to find the site at http://[sitename].local:9000.
7. And log into the container, should you need to, with `docker exec -it [sitename] bash`.  The site can be found under `/var/www/[sitename].`

# Affected versions or modules
- It affects the 8.x-1.0-alpha1 release of drupal-caravan: https://github.com/SU-SWS/drupal-caravan/releases/tag/8.x-1.0-alpha1.
- Please note areas for improvement here: https://docs.google.com/document/d/1DAoQHLwqRwjRmY-_oCjhcN1VJAritX-aaqXEK_ooMj8/edit#heading=h.4jeipn4huhwa.

# Associated Issues and/or People
- EARTH-260: https://stanfordits.atlassian.net/browse/EARTH-260
